packages/eslint-plugin-tslint/src/rules/config.ts

```diff
@@ -70,6 +70,8 @@ export default createRule<Options, MessageIds>({
     type: 'problem',
     messages: {
       failure: '{{message}} (tslint:{{ruleName}})`',
```

Comment:
This trailing ` on the message was accidentally added at some point, I will remove it once I have confirmed that the integration tests fail appropriately